nimbus-core/src/main/java/com/antheminc/oss/nimbus/domain/defn/ViewConfig.java

```diff
@@ -2359,6 +2359,69 @@ public static boolean isValidNumericFilter(FilterMode mode) {
 
 	}
 
+	/**
+	 * <p>RichText is a rich text editor.
+	 * 
+	 * <p><b>Expected Field Structure</b>
+	 * 
+	 * <p>RichText will be rendered when annotating a field nested under one of
+	 * the following components: <ul> <li>{@link Form}</li> </ul>
+	 * 
+	 * <p>RichText should decorate a field having a simple type.
+	 * 
+	 * @since 1.2
+	 */
+	@Retention(RetentionPolicy.RUNTIME)
+	@Target({ ElementType.FIELD })
+	@ViewStyle
+	public @interface RichText {
+
+		String alias() default "RichText";
+
+		/**
+		 * <p>Drives the size of the component by how many columns it occupies
+		 * on the UI.
+		 */
+		String cols() default "";
+
+		String controlId() default "";
+
+		/**
+		 * <p>CSS classes added here will be added to a container element
+		 * surrounding this component. <p>This can be used to apply additional
+		 * styling, if necessary.
+		 */
+		String cssClass() default "";
+
+		/**
+		 * <p>Whether or not this field should be considered a user-input field.
+		 */
+		boolean dataEntryField() default true;
+
+		/**
+		 * <p>Placeholder text to show when editor is empty.
+		 */
+		String placeholder() default "";
+
+		/**
+		 * <p>Whitelist of formats to display, see <a
+		 * href="http://quilljs.com/docs/formats/">here</a> for available
+		 * options.
+		 */
+		String formats() default "";
+
+		/**
+		 * <p>CSV separated string of toolbar features to include.
+		 */
+		String toolbarFeatures() default "header, font, bold, italic, underline, strike, color, background, script, size, blockquote, codeBlock, list, indent, direction, align, link, image, video, clean";
```

Comment:
Can we do ENUM for toolbar feature and have the attribute changed to an Array?

Reply:
Fixed.